### PR TITLE
solution

### PR DESCRIPTION
**Pull request by Anton-Zhytonbaiev**

### CODE REVIEW

**Review by OlhaYevstifieieva — APPROVED** [review]
GJ 🫂

File: src/sortStudents.ts [hunk 1]
```diff
@@ -1,16 +1,55 @@
...
+  Age = 'age',
+  Married = 'married',
+  AverageGrade = 'grades',
 }
 
-// create SortOrder type
-export type SortOrder;
+export type SortOrder = 'asc' | 'desc';
 
+const getAver = (marks:number[]): number => {
```
Comment (ArturErmolenko) [inline src/sortStudents.ts:20]: use better naming
```suggestion
const getAvaregeGrades = (marks:number[]): number => {
```

File: src/sortStudents.ts [hunk 2]
```diff
@@ -1,16 +1,55 @@
...
+  const averageMark:number = marks
+    .reduce((prev: number, curr: number): number => prev + curr, 0)
+    / marks.length;
```
Comment (ArturErmolenko) [inline src/sortStudents.ts:23]: 1. You can `return` immediately
2. `Array` is already typed above so you don't need type callback arguments

```suggestion
  return marks
    .reduce((acc, curr): number => acc + curr, 0)
    / marks.length;
```

File: src/sortStudents.ts [hunk 3]
```diff
@@ -1,16 +1,55 @@
...
 
-export function sortStudents(students, sortBy, order) {
-  // write your function
+  return averageMark;
```
Comment (ArturErmolenko) [inline src/sortStudents.ts:25]:
```suggestion
```

**Review by ArturErmolenko — CHANGES_REQUESTED** [review]
GJ 👍 
Left comments ⭐

**Review by ArturErmolenko — APPROVED** [review]
Well Done 👍

**Pull request closed without merging**